core/src/test/java/io/questdb/test/cutlass/pgwire/PGJobContextTest.java

```diff
@@ -2381,16 +2381,16 @@ public void testDropTable() throws Exception {
         String[][] sqlExpectedErrMsg = {
                 {"drop table doesnt", "ERROR: table does not exist [table=doesnt]"},
                 {"drop table", "ERROR: expected IF EXISTS table-name"},
-                {"drop doesnt", "ERROR: expected TABLE table-name or ALL TABLES, found unexpected [token='doesnt']"},
-                {"drop", "ERROR: expected TABLE"},
+                {"drop doesnt", "ERROR: 'table' expected"},
```

Comment:
This error doesn't seem to be the correct one as now we support DROP TABLE and DROP ALL TABLES. It should be enough to update the messages in the `unknownDropStatement` and `unknownDropTableSuffix` methods to keep the tests as they are.

Reply:
should have thought that!, done